### ISSUE TITLE
Pip install failing, python 3.9 unsupported?

### ISSUE THREAD

**Issue by masolimano** [astropy/regions#330]
Hi all,
I recently updated my python to the 3.9 version and I'm no longer able to install the regions package (v0.4). I tried
`pip install regions` and got:
```
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output
```

Full log here:
https://pastebin.com/xFXnxyAy

Can this be due to pyhon 3.9 being unsuppported in that release?

Cheers,
M

**Comment by astrofrog** [comment]
I ran into a similar issue, I think we need to release v0.5 which has infrastructure updates.

**Comment by astrofrog** [comment]
@larrybradley @keflavich - I'd be happy to trigger a release if you like? We have auto sdist- and wheel-building machinery set up in this repo.

**Comment by larrybradley** [comment]
@astrofrog 👍 I too was thinking it's time for a new release.

**Comment by larrybradley** [comment]
@astrofrog Please wait until next week for a new release.  I'm going to be making several updates this week.

**Comment by astrofrog** [comment]
Sounds good!

**Comment by avialxee** [comment]
Hey! how possible is a pre-release with the fix?

**Comment by larrybradley** [comment]
Regions v0.5 is released on PyPI, including wheels for Python 3.9.
